### DISCUSSION TITLE
how to rename "admin" folder

### DISCUSSION THREAD

**Discussion by Akila3** [YOURLS/YOURLS#3272] [Q&A] [answered]
how do I rename the "admin" folder to something else for security reasons on v1.8.2?
I tried just renaming the "admin" folder for example from: http://mydomain.dom/admin -> to -> http://mydomain.dom/kuiper, but I fail
to load the admin console with the new name. the server returned HTTP 404
had to rename it back again to "/admin" for it to load. it loads fine when the folder is "admin"
any ideas what else i need to do to rename the admin folder for it to work smoothly?

**Comment by ozh** [comment, marked as answer]
There's no need to post twice the same question if no one answers your question within a couple hours.

This topic is covered in various issues already. Short answer : you cannot. Less short answer : you can do it with a plugin. Long answer that would need a bit of googling on your end : this has not much to do with security as, would an exploit or bug exist in YOURLS, a whole bunch of other files are still at the same location so a hacker wouldn't care about your /admin folder location.
